src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2888,4 +2888,7 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Updating_the_attributes_of_0_is_not_supported_by_the_runtime" xml:space="preserve">
     <value>Updating the attributes of '{0}' is not supported by the runtime.</value>
   </data>
+  <data name="An_update_that_causes_the_return_type_of_implicit_main_to_change_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
+    <value>An update that causes the return type of the implicit Main method to change will prevent the debug session from continuing.</value>
```

Comment:
very very minor nit: IIRC, the compiler APIs don't consider the symbol as implicitly declared. I'd use the word "synthesized" instead of "implicit".

Reply:
I take your point, but I think to the end user probably neither "synthesized" nor "implicit" are too meaningful. Having said that I'll leave this for now as I expect we'll clean up all of these messages in a future PR, as "prevent the debug session from continuing" doesn't really make sense with Hot Reload, which might not even have a debug session.